crates/orderbook/openapi.yml

```diff
@@ -1187,6 +1187,7 @@ components:
               ZeroAmount,
               IncompatibleSigningScheme,
               TooManyLimitOrders,
+              TooMuchGas,
```

Comment:
@mfw78 this introduces a new error type the watchtower needs to know about to not get stuck indexing these orders, right?

Reply:
https://github.com/cowprotocol/cow-sdk/issues/202